c_plus_plus/src/model/audio/pulseaudio/pulseaudio_sink.cpp

```diff
@@ -0,0 +1,88 @@
+#include "pulseaudio_sink.h"
+
+#include <iostream>
+
+#include <pulse/error.h>
+
+
+namespace Model
+{
+
+PulseAudioSink::PulseAudioSink(const std::string& device,
+                               uint8_t channels,
+                               uint32_t rate,
+                               pa_sample_format format) :
```

Comment:
Unless there is a reason, I think we should put all these parameters as constants.